src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -207,8 +206,17 @@ private ProgressStepDefinition[] CreateWorkflowSteps(IProgressController control
             foreach (var language in languages)
             {
                 notifier.NotifyCurrentProgress(string.Format(CultureInfo.CurrentCulture, Strings.DownloadingQualityProfileProgressMessage, language.Name));
+
+                QualityProfile qualityProfileInfo;
+                if (!host.SonarQubeService.TryGetQualityProfile(this.connectionInformation, this.project, language.ServerKey, cancellationToken, out qualityProfileInfo))
+                {
+                    failed = true;
+                    break;
```

Comment:
Don't agree here, if you're saying that the plugin is not installed then in terms SQ the project needs to be excluded from analysis. We will fail in this case and which point the user will be able to exclude and retry.